build.gradle

```diff
@@ -126,7 +126,7 @@ subprojects { subproject ->
 		pahoMqttClientVersion = '1.2.0'
 		postgresVersion = '42.0.0'
 		reactorNettyVersion = '0.7.4.RELEASE'
-		reactorVersion = '3.1.4.RELEASE'
```

Comment:
Shall we wait for netty before merging?

Reply:
Right, I don't have any objects.
This is here just in case you would like to review.

Thanks